android/app/build.gradle

```diff
@@ -30,7 +30,7 @@ android {
 
     defaultConfig {
         applicationId "com.breez.client"
-        minSdkVersion 24
+        minSdkVersion 28
```

Comment:
@erdemyerebasmaz @roeierez The minimum Android version required from Nougat to Pie. Have you had a chance to talk about that?

Reply:
I did check the path provider, and it uses min SDK to 16 https://github.com/flutter/packages/blob/main/packages/path_provider/path_provider_android/android/build.gradle#L32 What makes sense; there is nothing special about its functionality to require API 28

Reply:
@ademar111190 Good points, most of the flutter plugins's minSDKVersion was set to flutter.minSDKVersion recently but even so, flutter.minSDKVersion is not 28 yet on Flutter 3.10. This was likely not a cache issue on my end from downgrading. (Still had the same error after repairing cache)

Let's discuss this and revert if possible on another issue.